docs/monitoring/metrics.md

```diff
@@ -335,6 +345,21 @@ Parameters:
 - `ttl` - time-to-live for transmitted UDP packets
 - `addressingMode` - UDP addressing mode to use (UNICAST/MULTICAST)
 
+Example configuration:
+
+{% highlight yaml %}
+
+metrics.reporters: gang
```

Comment:
there is no limit, but i wanted to make sure that users understand that the name is arbitrary; as in to configure graphite the reporter does not have to called "graphite" as well.